tooling/cli/scripts/kill-children.sh

```diff
@@ -0,0 +1,10 @@
+function getcpid() {
+    cpids=`pgrep -P $1|xargs`
+    for cpid in $cpids;
+    do
+        echo "$cpid"
+        getcpid $cpid
+    done
+}
+
+kill -9 $(getcpid $1)
```

Comment:
The change is you're calling 9 (SIGKILL) instead of SIGTERM, right? I'm not sure that's a great idea :/

Reply:
Let me try with SIGTERM again

Reply:
hmm I think there's no other way, either we leave Svelte users with this super annoying bug or we use SIGKILL

Reply:
I wouldn't want to risk using SIGKILL, could have some rough consequences since it takes away processes' ability to shut down properly :/

As Amr pointed out, the Svelte template seems like it's spawning it's child process incorrectly

Reply:
i'll just revert it, maybe there's a place for us to document it